styles/Home.module.scss

```
@@ -5,23 +5,10 @@
   height: 100vh;
   display: flex;
   justify-content: space-around;
-}
-
-.mainText {
-  display: flex;
-  flex-direction: column;
-  width: 45%;
-  text-align: center;
-}
-
-.homeCard {
-  border-radius: 18px;
-  width: 25%;
-  height: 350px;
-  //transition: transform 250ms ease-in;
-  transition: all 250ms ease;
-  &:hover {
-    transform: scale(1.01);
-    box-shadow: 0px 0px 34px 5px rgba(0, 0, 0, 0.407);
+  &__text {
+    display: flex;
+    flex-direction: column;
+    width: 45%;
+    text-align: center;
```

well done, dude! thats the usage of BEM css :heart: :rocket: 
```suggestion
  &__text {
    display: flex;
    flex-direction: column;
    width: 45%;
    text-align: center;
```